File: Aplikacija/frontend/models/porodilja.js
export default class Porodilja {
    constructor(
        id,
        imeIPrezime,
        datumRodjenja,
        email,
        lozinka,
        telefon,
        lbo,
        pocetakTrudnoce,
        sePorodila,
        jmbg,
        profilnaSlika,
        leci = []
    ) {
        this.id = id;
        this.imeIPrezime = imeIPrezime;
        this.datumRodjenja = datumRodjenja;
        this.email = email;
        this.lozinka = lozinka;
        this.telefon = telefon;
        this.lbo = lbo;
        this.pocetakTrudnoce = pocetakTrudnoce;
        this.sePorodila = sePorodila;
        this.jmbg = jmbg;
        this.profilnaSlika = profilnaSlika;
        this.leci = leci;
    }
}

File: Aplikacija/BACK/back/Models/Porodilja.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MyBackend.Models
{
    [Table("porodilja")]
    public class Porodilja
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("imeiprezime")]
        public string ImeIPrezime { get; set; } = string.Empty;

        [Column("datumrodjenja")]
        public DateTime DatumRodjenja { get; set; }

        [Column("email")]
        public string Email { get; set; } = string.Empty;

        [Column("lozinka")]
        public string Lozinka { get; set; } = string.Empty;

        [Column("telefon")]
        public string Telefon { get; set; } = string.Empty;

        [Column("lbo")]
        public string LBO { get; set; } = string.Empty;

        [Column("pocetaktrudnoce")]
        public DateTime PocetakTrudnoce { get; set; }

        [Column("seporodila")]
        public bool SePorodila { get; set; }

        [Column("jmbg")]
        public string JMBG { get; set; } = string.Empty;

        [Column("profilnaslika")]
        public string? ProfilnaSlika { get; set; }

        public ICollection<Leci>? Leci { get; set; }
    }
}
